pkg/controller/disruption/disruption.go

```diff
@@ -469,45 +483,45 @@ func (dc *DisruptionController) removeDb(obj interface{}) {
 		}
 		pdb, ok = tombstone.Obj.(*policy.PodDisruptionBudget)
 		if !ok {
-			klog.Errorf("Tombstone contained object that is not a pdb %+v", obj)
+			klog.Errorf("Tombstone contained object that is not a PDB %+v", obj)
```

Comment:
@pohly, logger.Error requires err, so how we should proceed with klog.Errorf if no err here?

Reply:
The `err` parameter may be nil: `logger.Error(nil, "Tombstone contained object that is not a PDB", "obj", obj)`.